utils/litemap/src/store/mod.rs

```diff
@@ -140,16 +143,6 @@ pub trait StoreIterableMut<'a, K: 'a, V: 'a>: StoreMut<K, V> + StoreIterable<'a,
     /// Returns an iterator that moves every item from this store.
     fn lm_into_iter(self) -> Self::KeyValueIntoIter;
 
-    /// Adds items from another store to the end of this store.
-    fn lm_extend_end(&mut self, other: Self)
-    where
-        Self: Sized,
-    {
-        for item in other.lm_into_iter() {
-            self.lm_push(item.0, item.1);
```

Comment:
If you really want to get rid of `lm_push`, then just rewrite this default impl of `lm_extend_end` to use `lm_insert` or make `lm_extend_end` not be default-impled.

Reply:
Made `lm_extend_end` not default impled.